gmagickpixel_methods.c

```diff
@@ -74,6 +74,26 @@ PHP_METHOD(gmagickpixel, setcolor)
 }
 /* }}} */
 
+/* {{{ proto GmagickPixel GmagickPixel::setColorCount(int colorCount)
+    Sets the color count associated with this color.
+*/
+PHP_METHOD(gmagickpixel, setcolorcount)
+{
+	php_gmagickpixel_object *internp;
+	long color_count;
```

Comment:
Should be zend_long because Windows 32bit is still a thing...tragically.

Also you probably noticed the tests failed due to the test file needed to be added to the package.xml